mangaki/mangaki/management/commands/morphing.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import os
+import json
+from math import floor
+import numpy as np
+from mangaki.settings import DATA_DIR
+from mangaki.algo.side import SideInformation
+
+
+def load_poster_tags():
+    side = SideInformation()
+    return side.nb_tags, side.T
+
+def distance(a, b):
+    dist = (b - a)
+    return np.linalg.norm(dist)
+
+def dist2seg(point, a, segment):
+    seg_norm = np.linalg.norm(segment)
+    current = point - a
+    projection = np.dot(current, segment)/seg_norm
+    dist2seg = distance(a, point)**2 - projection**2
```

Comment:
what is the guarantee that this value is positive? (just a comment)

Reply:
https://en.wikipedia.org/wiki/Cauchy–Schwarz_inequality

Reply:
put the link IN the code :p